cassandra-four-zero-bridge/src/main/java/org/apache/cassandra/spark/reader/SSTableCache.java

```diff
@@ -59,15 +64,32 @@ public class SSTableCache
                                                                                           propOrDefault("sbr.cache.stats.expireAfterMins", 60));
     private final Cache<SSTable, BloomFilter>                         filter = buildCache(propOrDefault("sbr.cache.filter.maxEntries", 16384),
                                                                                           propOrDefault("sbr.cache.filter.expireAfterMins", 60));
+    private final Cache<SSTable, Optional<CompressionMetadata>>       compressionMetadata = buildCache(propOrDefault("sbr.cache.compressionInfo.maxEntries", 128),
```

Comment:
Can you comment why the cache value is `Optional`?

Reply:
Now looking at the GitHub comment, I realized that it can be confusing. I meant to ask for adding java code comment to explain the cache value.